.github/pull_request_template.md

```diff
@@ -3,26 +3,22 @@
 - Any additional information or context which may help the reviewer
  
 ### Checklist
-Please ensure you have addressed all concerns below before marking a PR "ready for review" or before requesting a re-review:
+Please ensure you have addressed all concerns below before marking a PR "ready for review" or before requesting a re-review. If you cannot complete an item below, replace the checkbox with the ⚠️ `:warning:` emoji.
```

Comment:
Oy - I was about to approve this, but realized that we should require an explanation in addition to just the ⚠️ . So something like:
"...replace the checkbox with the :warning: emoji and explain why the step was not completed."

Reply:
updated